Centaurus.Domain/Quanta/Handlers/QuantumHandler.cs

```diff
@@ -176,6 +178,16 @@ async Task<ResultMessage> AuditorHandleQuantum(MessageEnvelope envelope)
                 return result;
         }
 
+        void ValidateAccountRequestRate(MessageEnvelope envelope)
+        {
+            var request = envelope.Message as RequestQuantum;
+            if (request == null)
+                return;
+            var account = Global.AccountStorage.GetAccount(request.RequestMessage.Account);
+            if (!account.RequestCounter.IncRequestCount(request.Timestamp, out string error))
+                throw new TooManyRequests($"Too many quanta from account {account.Account.Pubkey.ToString()}.");
```

Comment:
"Too many quanta" is too vague. Something like "Request limit reached for account {account}" will be more specific. Also, I noticed that you never use the returned error message form the `IncRequestCount`. Either use it or drop this argument in the method definition.